packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -1191,11 +1194,31 @@ enum ResizeImagePolicy {
 /// Instructs Flutter to decode the image at the specified dimensions
 /// instead of at its native size.
 ///
+/// The [width] and [height] parameters refer to logical pixels.
+///
```

Comment:
Hey! Can someone confirm if it's logical pixels rather than physical pixels? The original issue seems to describe it differently. I’m not too familiar with this, but since the width and height are being used for decoding images, shouldn't physical pixels be used instead? Appreciate any clarification in advance!